ccpp/suites_not_used/suite_FV3_HAFS_v1_thompson_noahmp_nonsst.xml

```diff
@@ -18,7 +18,6 @@
       <scheme>rad_sw_pre</scheme>
```

Comment:
There is a plan to delete the suites under suites_not_used, is this xml still used in HAFS experiments?

Reply:
@junwang-noaa the suite_FV3_HAFS_v1_thompson_noahmp_nonsst.xml and suite_FV3_HAFS_v1_thompson_noahmp.xml are most likely needed in 2024 HAFS real-time parallel experiment configurations. In this case, not sure what to do, shall we move them out from the suites_not_used dir now?

Reply:
In any case, we will keep a note, probably move them out in a future PR, before them being deleted. Thanks!

Reply:
Yes, I suggest moving them out from the "suites_not_used" in the next fv3 PR. Thanks